implemented new background design

sabinagrisi opened the pull request:
liked the idea of the game being played on a wooden table, so that's that, and I also decided to use colors instead of an image because it makes more sense in terms of screen size, but it can always look prettier, so any criticisms? 

Will keep implementing new stuff (like color palette, new container design) later.

Commit:
implemented new background design

liked the idea of the game being played on a wooden table, so that's that, and I also decided to use colors instead of an image because it makes more sense in terms of screen size, but it can always look prettier, so any criticisms? Will keep implementing new stuff (like color palette, new container design) later.

bloomask commented:
I really like the idea of the wooden table. I suggest adding some more elements on the "table" around, to make it more understandable, because at first look it might be a bit hard to understand what it is.

Also, (this is for another PR) maybe making the middle part grid sheet notebook-inspired could be a fun touch of detail as well!

sabinagrisi commented:
Noted on the wooden table.

Yesss, the notebook one was exactly what I was thinking about as well! Working on it

Commit:
changed color palette for day mode

Changed it from a white to a light cream, loved the original orange for the buttons, only changed the tonality. And also another shade of dark gray for the text.

Eliminated unnecesary lines as well.

Will update the dark mode one after fixing the toggle theme.

Commit:
added cute pens for the table

There was a strange bug with the width in .ballpoint, where the given value was decreased by 25% whenever I played the game.  From 20px tp 15px, from 40px to 30px, from 10px to 8px...

After 4 hours, the solution was simply changing it to min-width: 20px;

Commit:
added dark-mode color palette

Added colors, moved every body.dark-mode to the bottom and simplified it, and also made turnStatusDisplay to always be white.

Furthermore, eliminated transform and box-shadow from button:disabled, to make it look more unclickable.

bloomask commented:
Looking very good already, ping me whenever it's ready to be merged :) @sabinagrisi

Commit:
added notebook-like design

for instructions, menu, container, and chatbox.

Commit:
added a cute handwriting font

bloomask committed:
Merge branch 'main' into UI-design
